src/test/java/teammates/test/cases/action/InstructorCourseEnrollSaveActionTest.java

```diff
@@ -243,9 +240,11 @@ public void testExecuteAndPostProcess() throws Exception {
         verifyNoTasksAdded(enrollAction);
         
         //fail to enroll, if exceed the range
-        enrollStringBuilder.append(Const.EOL).append("section" + sizeLimitBoundary + "\tteam" + sizeLimitBoundary
-                                                     + "\tname" + sizeLimitBoundary + "\temail" + sizeLimitBoundary
-                                                     + "@nonexistemail.nonexist");
+        enrollStringBuilder.append(Const.EOL).append(
+                "section" + Const.SIZE_LIMIT_PER_ENROLLMENT + "\tteam" + Const.SIZE_LIMIT_PER_ENROLLMENT
+                 + "\tname" + Const.SIZE_LIMIT_PER_ENROLLMENT + "\temail"
+                 + Const.SIZE_LIMIT_PER_ENROLLMENT
+                 + "@nonexistemail.nonexist");
```

Comment:
Try to fit these parameters onto as little lines as possible (as long as they don't exceed the character limit).